tests/integration-tests/tests/common/utils.py

```diff
@@ -90,11 +90,56 @@ def retrieve_latest_ami(region, os, ami_type="official", architecture="x86_64"):
         sys.exit(1)
 
 
+def retrieve_pcluster_ami_without_standard_naming(region, os, version="2.8.1", architecture="x86_64"):
+    try:
+        client = boto3.client("ec2", region_name=region)
+        ami_name = f"pcluster-without-standard-naming-{version}-{os}"
```

Comment:
How about a more descriptive name, something like `ami-for-test_create_wrong_pcluster_version-DO-NOT-DELETE`?